.github/PULL_REQUEST_GUIDELINES/rule_new_guidelines.md

```diff
@@ -0,0 +1,31 @@
+## Rule: New - Guidelines
+
+These guidelines serve as a reminder set of considerations when proposing a new rule.
+
+### Documentation and Context
+
+- [ ] Detailed description of the rule.
+- [ ] List any new fields required in ECS/data sources.
+- [ ] Link related issues or PRs.
+- [ ] Include references.
+
+### Rule Metadata Checks
```

Comment:
Some of the fields listed here are Rule Metadata and the others are Rule Data, do we want to separate them. Example - `note`, `setup`, `tags` `query`. If we want to consolidate into one, then best not to leave out `index` here. It will also depend on the rule type but unit tests for schemas will fail if anything.

Reply:
I think it makes sense to add in `index`. I tried to keep "Metadata" (capital M) as simple/inclusive as possible, not necessarily the same as out "metadata" or "rule" section.

Reply:
Cool, adding index then makes sense.